src/main/java/igrad/logic/commands/requirement/RequirementAddCommand.java

```diff
@@ -4,25 +4,23 @@
 import static igrad.logic.parser.CliSyntax.PREFIX_TITLE;
 import static java.util.Objects.requireNonNull;
 
-import java.util.ArrayList;
+import java.util.List;
 
 import igrad.logic.commands.CommandResult;
 import igrad.logic.commands.exceptions.CommandException;
 import igrad.model.Model;
 import igrad.model.requirement.Requirement;
 import igrad.model.requirement.RequirementCode;
-import javafx.collections.ObservableList;
 
 /**
  * Adds a requirement to the course.
  */
 public class RequirementAddCommand extends RequirementCommand {
     public static final String COMMAND_WORD = REQUIREMENT_COMMAND_WORD + SPACE + "add";
 
-    public static final String MESSAGE_DETAILS = COMMAND_WORD + ": Adds a requirement with relevant details "
-        + "specified.\n";
+    public static final String MESSAGE_DETAILS = COMMAND_WORD + ": Adds a requirement.\n";
 
-    public static final String MESSAGE_USAGE = "Parameter(s): "
+    public static final String MESSAGE_USAGE = "Parameters: "
```

Comment:
ok since you preferred this way, i shall change modules too, to make it consistent